docs/content/modeling/roles-and-permissions.mdx

```diff
@@ -156,13 +155,13 @@ Relating roles within <ProductName format={ProductNameFormat.ShortForm}/> can be
   }}
 />
 
-### 02. Adding Permissions For Bookings
+### 02. Add Permissions For Bookings
 
```

Comment:
https://oktawiki.atlassian.net/wiki/spaces/DOCS/pages/2544472397/Document+Structure#Capitalization

Reply:
Per @rhamzeh, if we would like to reference a generalized "Best practices" ... I would refer here:

https://developers.google.com/style/headings?hl=en#heading-and-title-format

Essentially, we should be using sentence case for headers. Let me know if you have any questions. Thanks!